annotation.go

```diff
@@ -81,11 +81,14 @@ type AnnotationService interface {
 	// ListStreams lists all streams matching the filter.
 	ListStreams(ctx context.Context, orgID platform.ID, filter StreamListFilter) ([]ReadStream, error)
 	// CreateOrUpdateStream creates or updates the matching stream by name.
-	CreateOrUpdateStream(ctx context.Context, stream Stream) (*ReadStream, error)
+	CreateOrUpdateStream(ctx context.Context, orgID platform.ID, stream Stream) (*ReadStream, error)
```

Comment:
Needs the `orgID` here for the case of creating a stream - the backend needs to know what org to create the stream for.